nemo/collections/asr/models/asr_model.py

```diff
@@ -47,3 +51,61 @@ def multi_test_epoch_end(self, outputs, dataloader_idx: int = 0):
         wer_denom = torch.stack([x['test_wer_denom'] for x in outputs]).sum()
         tensorboard_logs = {'test_loss': val_loss_mean, 'test_wer': wer_num / wer_denom}
         return {'test_loss': val_loss_mean, 'log': tensorboard_logs}
+
```

Comment:
Doesnt ASRModel need to extend Exportable?

Reply:
I am not quite happy with current approach to Model vs. Exportable "interface", actually. 
Current default implementation of Exportable mixin assumes Module and does not even always fit Model. 
Return value of export() is PT/ONNX graph for the module and 2 lists (file names and doc strings) for Model.
I believe Model baseclass should have export() method (returning 2 lists) that throws NotImplemented by default; the models that do implement export() can use additional mixins or utility functions to share export code patterns.

Reply:
This makes sense, Exportable was ported by taking Nemo 0.x export example - which were at module level only, so the Exportable interface was designed to support module level export as well. 

I'm ok with the ASR side changes, just needs review by others to make sure export works properly.